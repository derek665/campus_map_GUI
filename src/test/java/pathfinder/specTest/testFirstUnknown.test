# test correct output when the first node is unknown

CreateGraph testFirst

AddNode testFirst B

FindPath testFirst A B