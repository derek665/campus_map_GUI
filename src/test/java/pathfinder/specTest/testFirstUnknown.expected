# test correct output when the first node is unknown

created graph testFirst

added node B to testFirst

unknown node A